rust-enjoyer/.cargo/config.toml

```diff
@@ -0,0 +1,2 @@
+[build]
+rustflags = ["-C", "link-arg=-fuse-ld=lld", "-C", "target-cpu=native"]
```

Comment:
Some how i get a: 
```
error: could not compile `rustix` (build script) due to previous error
error: linking with `cc` failed: exit status: 1
  |
  = note: LC_ALL="C" PATH="/root/.rustup/toolchains/ ...
```

Changing this to:
```
rustflags = ["-C", "link-arg=-fuse-ld=lld", "-C", "target-cpu=native"]
```
fixes it for me.

Reply:
sorry I should have removed that -- `lld` has slightly faster compile times for development, but not safe to assume everyone has it installed (or require them to do so)